image_provisioner/playbooks/build_ami.yaml

```diff
@@ -76,7 +86,7 @@
       region: "{{ region }}"
       state: present
       description: This was provisioned {{ ansible_date_time.iso8601 }}
-      name: ocp-{{ openshift_version }}-{{ ami_type }}-gold-auto
+      name: ocp-{{ openshift_version }}-{{ ami_type }}-gold-auto-missing-logging-fluentd
       wait: yes
```

Comment:
@chaitanyaenr, you are going to remove the "-missing-logging-fluentd" from ami name ?

Reply:
Fixed.